src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj

```diff
@@ -1,11 +1,13 @@
 <Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
   
   <PropertyGroup>
-    <TargetFrameworks>xamarinmac20;xamarinios10;MonoAndroid12.0;net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>xamarinmac20;xamarinios10;MonoAndroid12.0;netstandard2.0;net461</TargetFrameworks>
 		<TargetFrameworksCI>MonoAndroid11.0;MonoAndroid12.0;xamarinios10;net461;netstandard2.0;xamarinmac20</TargetFrameworksCI>
   </PropertyGroup>
 	
-	<Import Project="../targetframework-override.props" />
+	<PropertyGroup Condition="'$(UnoTargetFrameworkOverride)'!=''">
+		<TargetFrameworks>$(UnoTargetFrameworkOverride)</TargetFrameworks>
+	</PropertyGroup>
```

Comment:
This is very curious, the `	<Import Project="../targetframework-override.props" />` is doing exactly this. What's the error in vs4mac without this ?

Reply:
VS4M is unable to load the dependencies

Reply:
@jeromelaban this is the specifc error on the Package Console:

`/usr/local/share/dotnet/sdk/6.0.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: android [/Users/apineda/develop/uno/uno/src/SourceGenerators/System.Xaml/Uno.Xaml.net6.csproj]

/usr/local/share/dotnet/sdk/6.0.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To install these workloads, run the following command: dotnet workload install android [/Users/apineda/develop/uno/uno/src/SourceGenerators/System.Xaml/Uno.Xaml.net6.csproj]

/Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/MSBuild/Current/bin/NuGet.targets(352,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/Users/apineda/develop/uno/uno/src/Uno.UI.sln]

MSBuild exited with code 1
`

Reply:
Interestingly, this means that uno-check needs to be executed to install the proper sdks. Can you try doing that ?

Reply:
Sure!

Reply:
@jeromelaban  ran the `uno-check` and this fixed a few issues on my environment but I am still seeing the same error reported above when using the `<Import Project...`

Reply:
Got it. Let's merge it as-is then, thanks!